javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+const $countdownDisplay = document.querySelector('.countdown-display');
+let count = 4;
+function updateHeader() {
+  $countdownDisplay.textContent = count;
+
+  if (count === 0) {
+    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(intervalID);
+  }
+  count--;
+}
+
+const intervalID = setInterval(updateHeader, 1500);
```

Comment:
Because the countdown should decrease one per second the miliseconds should be `1000` not `1500`.